tools/toolchains/arm.py

```diff
@@ -66,6 +66,9 @@ def __init__(self, target, notify=None, macros=None,
         self.asm = [main_cc] + self.flags['common'] + self.flags['asm']
         self.cc = [main_cc] + self.flags['common'] + self.flags['c']
         self.cppc = [main_cc] + self.flags['common'] + self.flags['c'] + self.flags['cxx']
+        if target.name in ["REALTEK_RTL8195AM"]:
```

Comment:
Could you not add 
```C
    MAX_ENUM_VALUE=0XFFFFFF
```
to the enums that must be 4bytes wide in mbed-os? Then you would not be breaking the portability of code. @pan- @geky Would this work well (with suitably different names per-enum)?